From 0d588211db60fc5274f8bd607960fcd4fa3de511 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sun, 16 Aug 2015 16:14:49 +0000
Subject: [PATCH 1/3] Fix compilation when WINVER is set to 0x600 or higher

We define the `MARGINS` data structure explicitly if the WINVER
is lower than Windows Vista. However, when WINVER corresponds to Windows
Vista or later, we need to `#include <uxtheme.h>` for the `MARGINS` data
structure to be defined.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 src/winmain.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/winmain.c b/src/winmain.c
index a4bc484..1e68dcf 100644
--- a/src/winmain.c
+++ b/src/winmain.c
@@ -43,6 +43,10 @@ typedef struct {
   int cyBottomHeight;
 } MARGINS;
 
+#else
+
+#include <uxtheme.h>
+
 #endif
 
 static HRESULT (WINAPI *pDwmIsCompositionEnabled)(BOOL *);
-- 
2.5.0

